app/controllers/feedback_controller.rb

```diff
@@ -31,6 +32,6 @@ def verify_access
   def feedback_params
     params.require(:feedback)
           .permit(:feedback, :contact_email)
-          .merge(application: session[:redirect], username: current_user.display_name)
+          .merge(application: app_display_name, username: current_user.display_name)
```

Comment:
we can put this off to a PR later — but i think we should change the name of this param to `subject`, since we will want to collect feedback from different parts of the application?